docs/sdk/android-async.md

```diff
@@ -29,6 +32,14 @@ future.thenAccept(new AppCenterConsumer<{ReturnType}>() {
     }
 });
 ```
+```kotlin
+val future = {AnyAsyncApi}()
+future.thenAccept(object : AppCenterConsumer<{ReturnType}> {
+    override fun accept(t: {ReturnType}?) {
+        // do something with result, this is called back in U.I. thread.
+    }
+})
```

Comment:
Not an issue per se but I find it quite odd that we don't mention that the code block above is pseudo-code. Maybe we call this out explicitly?

Reply:
Made a small remark.